GitHub-Actions/two-environment-pipeline-template/questions.json

```diff
@@ -1,5 +1,9 @@
 {
   "questions": [{
+    "key": "intro",
+    "question": "\nThis template configures a pipeline that deploys a serverless application to a testing and a production environments.\n",
+    "kind": "info"
+  }, {
     "key": "pipeline_user_aws_access_key_id_variable_name",
     "question": "What is the GitHub Secret name for pipeline user account access key id?",
```

Comment:
```suggestion
    "question": "What is the GitHub secret name for pipeline user account access key ID?",
```

Reply:
Actually this applies to all instances:

- Should be "GitHub secrets"; it's not used as a proper noun.
- ID should be all caps.
- Git should be capitalized (not "git").